queue/src/fiskaltrust.Interface.Tagging/AT/Models/ftChargeItemCaseFlagsATV1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace fiskaltrust.Interface.Tagging.AT.Models
+{
+    public enum ftChargeItemCaseFlagsATV1 : long
```

Comment:
I'm a fan of only using namespaces to distinguish different things with the same name 🤔
like with the `ReceiptRequest` we also have two different versions `V0` and `V1` both called `ReceiptRequest` but in different namespaces `fiskaltrust.ifPOS.v0.ReceiptRequest` and `fiskaltrust.ifPOS.v1.ReceiptRequest`.

you can just have a `using fiskaltrust.ifPOS.v1` if you only need one or if you need both `using v1 = fiskaltrust.ifPOS.v1` and `using v0 = fiskaltrust.ifPOS.v0` and then use them with `new v1.ReceiptRequest()` or `new v0.ReceiptRequest()`;

We could use the same pattern here 😁 Don't have The market or the version in the Model names and just put them in the correct namespaces like this

```suggestion
﻿namespace fiskaltrust.Interface.Tagging.Models.V1.AT
{
    public enum ftChargeItemCaseFlags : long
```

and for v2 like this

```
﻿namespace fiskaltrust.Interface.Tagging.Models.V2
{
    public enum ftChargeItemCaseFlags : long
```

what do you think?

Reply:
its applied